src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -216,6 +220,14 @@ def _taskrun_format_group(item):
                                    _get_value(item, 'runResult', 'finishTime')))
     ])
 
+def _agentpool_format_group(item):
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('STATE', _get_value(item, 'provisioningState')),
+        ('OS', _get_value(item, 'os')),
```

Comment:
os is in properties, not in root. So you might need write as ('OS', _get_value(item, 'properties', 'os')),

Reply:
Right now is working, I do see that it shows as part of the properties but when I call the get method in sdk I get this response. 
{
  "count": 1,
  "id": "/subscriptions/3647315e-0c5b-4ce4-8739-b071e144b2c9/resourceGroups/roywusdf/providers/Microsoft.ContainerRegistry/registries/royagent/agentPools/rosanchagentpool",
  "location": "WESTUS",
  "name": "rosanchagentpool",
  "os": "Linux",
  "provisioningState": "Succeeded",
  "resourceGroup": "roywusdf",
  "tags": null,
  "tier": "S1",
  "type": "Microsoft.ContainerRegistry/registries/agentPools",
  "virtualNetworkSubnetResourceId": null
}